bookshelf/app.yaml

```diff
@@ -0,0 +1,12 @@
+runtime: custom
+vm: true
+
+manual_scaling:
```

Comment:
@tmatsuo I think we should remove it for simplicity and clarity, but also include a section in getting started explaining the situation. That way, the users start simple and we use it as a teaching tool:

# Manual Scaling
By default managed VMs will deploy to 2 instances behind a load balancer. To override this, which can be preferable for this free trial because it reduces the cost of running this application, add the following to `app.yaml`:

```yaml
manual_scaling:
  instances: 1
```